assets/styles/alps-theme.scss

```diff
@@ -11,3 +11,65 @@
      border-bottom: 1px solid lighten(#000, 90);
    }
  }
+
+ .widget {
+   ul li {
+     margin-bottom: 10px;
+     a {
+       font-family: "Merriweather",Georgia,Times,"Times New Roman",serif;
```

Comment:
Since this is all new CSS specifically for the ALPS WP theme, we might want to start to create a mini-ALPS framework here. 

This is the ALPS repo (it's public) - https://github.com/adventistchurch/alps

It might be helpful to copy some (not all) settings [from here](https://github.com/adventistchurch/alps/blob/master/source/css/_settings.variables.scss) so we're not repeating ourselves in case this file grows any larger.